tectonic/core-bridge.c

```diff
@@ -69,7 +71,7 @@ dvipdfmx_simple_main(tt_bridge_api_t *api, char *dviname, char *pdfname)
 {
     extern int dvipdfmx_main(int argc, char *argv[]);
 
-    char *argv[] = { "dvipdfmx", "-o", pdfname, dviname };
+    char *argv[] = { xstrdup("dvipdfmx"), xstrdup("-o"), pdfname, dviname };
```

Comment:
It is not possible to use a const. `dvipdfmx_main` calls `do_early_args` and `do_args`, which call `getopt_long`. This function requires a `char * const argv[]`.

The only solution would be to avoid the allocation would be to do an explicit cast. I'm not sure this is the best approach. A better approach would be to fix the code on its own and not have a function that expect `argc` and `argv` (I think such significant change belongs to another PR).

Hence, my solution was to use `xstrdup` which is used elsewhere in such a case. I don't think two additional allocations will have an impact on the performance. It also fixes the issue without using an explicit cast that hide the issue (i.e., any function inside `dvipdfmx_main` could still try to modify the content of `argv[0]` or `argv[1]` resulting in a crash).